apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/SimpleManagedTracer.cs

```diff
@@ -19,7 +19,9 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 
-namespace Google.Devtools.AspNet
+using CloudTrace = Google.Cloud.Trace.V1.Trace;
```

Comment:
I was unable to make this work.  When the name space goes to Google.Cloud.* it starts to confused the namespace Trace with the class Trace even if I alias it.

I tried to figure this out but couldn't find a solution